arm-hdinsight/2015-03-01-preview/swagger/cluster.json

```diff
@@ -404,6 +404,9 @@
                 },
                 "configurations": {
                     "type": "object",
+                    "additionalProperties": {

```

Comment:
For example, this code worked before:

```csharp
clusterDefinition.configurations = new { A = new { C = "Hello!" } };
```

but it will be a compilation error now.